src/jdk.incubator.concurrent/share/classes/jdk/incubator/concurrent/StructuredTaskScope.java

```diff
@@ -505,7 +505,7 @@ private void implJoin(Duration timeout)
 
     /**
      * Wait for all threads to finish or the task scope to shut down. This method waits
-     * until all threads started in the task scope finish execution (of both task and
+     * until all threads contained in the task scope finish execution (of both task and
```

Comment:
I checked the usages of "started in" and "contained in" in the javadoc and I think they are consistently used. The phrase "contained in" is introduced in the context of confinement and then used in the description and exceptions for methods that are confined to threads in the tree. In the case of the join methods, they wait for the threads started directly in the task scope and so using "started in" is correct. Waiting for the threads started in the task scope does imply waiting indirectly for all threads in the tree and we could potentially expand the javadoc on this point.

Reply:
Thanks Alan. I just opened https://github.com/openjdk/loom/pull/195 to fix the typo and will make an attempt at expanding the Javadoc once I have a better understanding of things.